operator/deploy/cluster_role_binding.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 subjects:
 - kind: ServiceAccount
   name: cass-operator
-  namespace: ""
+  namespace: "cass-operator"
```

Comment:
I thought this was a mistake when I was reviewing this, but it's just unclear what should be here

Reply:
FYI: This works for my integration test with a namespace of "".

Reply:
I think the code that needs this ClusterRoleBinding fails and continues - could be the explanation?